dataprofiler/data_readers/csv_data.py

```diff
@@ -625,7 +625,7 @@ def is_match(cls, file_path, options=None):
                 count = len([i.start() for i in re.finditer(delimiter_regex, line)])
             else:
                 # If no delimiter, see if spaces are regular intervals
-                count = len([i.start() for i in re.finditer(space_regex, line)])
+                count = len([i.start() for i in re.finditer(space_regex, line.rstrip())])
```

Comment:
fixes issue with delimiters being counted when empty to its right.

Reply:
Could it be empty to its left?

Reply:
did we have the test for this one?

Reply:
added

Reply:
still no general stripping? `line.strip()`

Reply:
I believe we are only addressing the case where it is a bunch of whitespace tacked on to the end, if we did both sides it might mess up the count.

Reply:
I see, I don't know if pulling data from the right is appropriate either? 

Was there a particular error that occurred?

Reply:
Yes, specifically the test I added. where it was a column of numbers but an extra space existed after the numbers